stories/Divider.stories.mdx

```diff
@@ -0,0 +1,117 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+import { moduleMetadata } from '@storybook/angular';
+import { DividerComponent } from '../projects/ion/src/lib/divider/divider.component';
+
+<Meta title="Ion/Dividers" component={DividerComponent} />
+
```

Comment:
```suggestion

export const Template = (args) => ({ props: args });

```
You can add this template and use it with your stories, just adding args as a prop on Story.
This will allow us to render controls correctly and change values while on Storybook.